Jenkinsfile

```
@@ -21,26 +21,41 @@ nodeWithTimeout('docker') {
             sh 'make shellcheck'
         }
 
-        /* Outside of the trusted.ci environment, we're building and testing
-         * the Dockerfile in this repository, but not publishing to docker hub
-         */
-        stage('Build') {
-            sh 'make build'
-        }
-
-        stage('Prepare Test') {
-            sh "make prepare-test"
-        }
-
-        def labels = ['debian', 'slim', 'alpine', 'jdk11', 'centos', 'centos7']
+        def configs = [
+                      'amd64' : ['debian', 'slim', 'alpine', 'jdk11', 'centos', 'centos7'],
+                      'arm64' : ['debian', 'slim', 'alpine', 'jdk11'],
+                      's390x' : ['debian', 'slim', 'alpine', 'jdk11'],
+                      'ppe64le' : ['debian', 'slim', 'alpine', 'jdk11']
```

We'll need to leave this as a comment until we resolve the issue with the ppc64le agent on ci.jenkins.io.
```suggestion
                      // 'ppe64le' : ['debian', 'slim', 'alpine', 'jdk11']
```